content/en/blog/_posts/2021-08-20-seccomp-default.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: blog
+title: "Hardening Kubernetes security by enabling seccomp by default"
+date: 2021-08-20
+slug: seccomp-default
+---
+
+**Author:** Sascha Grunert, Red Hat
+
+This blog post is about a new Kubernetes feature introduced in v1.22, which adds
+an additional security layer on top of the existing seccomp support. Seccomp is
+a security mechanism for Linux processes to filter system calls (syscalls) based
+on a set of defined rules. Applying seccomp profiles to containerized workloads
+is one of the key tasks when it comes to enhancing the security of the
+application deployment. Developers, site reliability engineers and
+infrastructure administrators have to work hand in hand to create, distribute
+and maintain the profiles over the applications life-cycle.
+
+The [`SecurityContext`][seccontext] of Pods and Containers can be used to adjust
```

Comment:
_nit_
We usually write field names how they appear in the API rather than how they appear in source code, so:
```suggestion
The [`securityContext`][seccontext] of Pods and their containers can be used to adjust
```
_or_
```suggestion
You can use the [`securityContext`][seccontext] field of Pods and their containers to adjust
```

Reply:
Sounds good, changed as suggested :+1: